.github/workflows/publish_package.yml

```diff
@@ -7,8 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Install Node version from nvm
-      run: echo ::set-output name=NVMRC::$(cat .nvmrc)
-      id: nvm install
+      run: nvm install
```

Comment:
If we're using nvm to install Node why do we need `actions/setup-node` later?

Reply:
Oh yeah, of course

Reply:
Hmm, on further investigation it seems to be possible to also do it using `npm config set registry <registry url>` but we can try this first and see if it behaves as hoped.